pkg/authorization/authorizer/authorizer.go

```diff
@@ -27,17 +27,17 @@ func NewSubjectLocator(delegate authorizerrbac.SubjectLocator) SubjectLocator {
 	return &openshiftSubjectLocator{delegate: delegate}
 }
 
-func (a *openshiftAuthorizer) Authorize(attributes authorizer.Attributes) (bool, string, error) {
+func (a *openshiftAuthorizer) Authorize(attributes authorizer.Attributes) (authorizer.Decision, string, error) {
 	if attributes.GetUser() == nil {
-		return false, "", errors.New("no user available on context")
+		return authorizer.DecisionDeny, "", errors.New("no user available on context")
```

Comment:
mechanical change to return NoOpinion, not Deny

Reply:
> mechanical change to return NoOpinion, not Deny (applies everywhere)

done